Update based on legal requirements. (#900)
huggingface/optimum-habana: docs/source/concept_guides/hpu.mdx

limitations under the License.
-->

> # What are Intel® Gaudi® accelerator first-generation Gaudi, Gaudi2 and HPUs?

Reviewer: ```suggestion
# What are Intel Gaudi 1, Intel Gaudi 2 and HPUs?
```

> [Gaudi](https://habana.ai/training/gaudi/) and [Gaudi2](https://habana.ai/training/gaudi2/) are the first- and second-generation AI hardware accelerators designed by Habana Labs.

Reviewer: ```suggestion
[Intel® Gaudi® 1](https://habana.ai/training/gaudi/) and [Intel® Gaudi® 2](https://habana.ai/training/gaudi2/) are the first- and second-generation AI hardware accelerators designed by Habana Labs and Intel.
```

limitations under the License.
-->

> # What are Intel Gaudi 1, Intel Gaudi 2 and HPUs?

Reviewer 2: ```suggestion
# What are Intel® Gaudi® 1, Intel® Gaudi® 2 and HPUs?
```

> [Intel® Gaudi® 1](https://habana.ai/training/gaudi/) and [Intel® Gaudi® 2](https://habana.ai/training/gaudi2/) are the first- and second-generation AI hardware accelerators designed by Habana Labs and Intel.

Reviewer 2: ```suggestion
[Intel Gaudi 1](https://habana.ai/training/gaudi/) and [Intel Gaudi 2](https://habana.ai/training/gaudi2/) are the first- and second-generation AI hardware accelerators designed by Habana Labs and Intel.
```

- *Eager mode* execution, where the framework executes one operation at a time as defined in [standard PyTorch eager mode](https://pytorch.org/tutorials/beginner/hybrid_frontend/learning_hybrid_frontend_through_example_tutorial.html).
- *Lazy mode* execution, where operations are internally accumulated in a graph. The execution of the operations in the accumulated graph is triggered in a lazy manner, only when a tensor value is required by the user or when it is explicitly required in the script. The [SynapseAI graph compiler](https://docs.habana.ai/en/latest/Gaudi_Overview/SynapseAI_Software_Suite.html#graph-compiler-and-runtime) will optimize the execution of the operations accumulated in the graph (e.g. operator fusion, data layout management, parallelization, pipelining and memory management, graph-level optimizations).

> See [here](../usage_guides/accelerate_training#lazy-mode) how to use these execution modes in 🤗 Optimum Intel(R) Gaudi(R) accelerator.

Reviewer 2: ```suggestion
See [here](../usage_guides/accelerate_training#lazy-mode) how to use these execution modes in 🤗 Optimum for Intel(R) Gaudi(R) accelerator.
```

> See [here](../usage_guides/accelerate_training#lazy-mode) how to use these execution modes in 🤗 Optimum for Intel® Gaudi® accelerator.

Reviewer: ```suggestion
See [here](../usage_guides/accelerate_training#lazy-mode) how to use these execution modes in Optimum for Intel Gaudi.
```
